plugins/gaming.py

```diff
@@ -55,25 +56,25 @@ def n_rolls(count, n):
 @hook.command("roll", "dice")
 def dice(text, notice):
     """<dice roll> - simulates dice rolls. Example: 'dice 2d20-d5+4 roll 2': D20s, subtract 1D5, add 4
+
     :type text: str
     """
-
     if hasattr(text, "groups"):
```

Comment:
This check shouldn't be needed anymore as `dice` is never called with a regex match